public_html/wp-content/plugins/pattern-directory/includes/pattern-flag-post-type.php

```diff
@@ -136,12 +136,12 @@ function check_flag_threshold( $post_ID, $post, $update ) {
 		'post_status' => PENDING_STATUS,
 	) );
 
-	$threshold = 3;
+	$threshold = absint( get_option( 'wporg-pattern-flag_threshold' ) );
```

Comment:
```suggestion
	$threshold = absint( get_option( 'wporg-pattern-flag_threshold', 5 ) );
```

I'm not sure if the default is set by the register_setting, but just in case :)